sass/components/forms/_switches.scss

```diff
@@ -56,36 +56,40 @@
     transition: left 0.3s ease, background .3s ease, box-shadow 0.1s ease, transform .1s ease;
   }
 
-  &:before {
-    background-color: transparentize($switch-bg-color, .85);
-  }
-
   &:after {
-    background-color: $switch-unchecked-bg;
+    background-color: $switch-thumb-unchecked-color;
     box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2), 0px 2px 2px 0px rgba(0, 0, 0, 0.14), 0px 1px 5px 0px rgba(0, 0, 0, 0.12);
   }
 }
 
+input[type=checkbox]:not(:disabled) ~ .lever:active:before,
+input[type=checkbox]:not(:disabled).tabbed:focus ~ .lever::before,
+input[type=checkbox]:not(:disabled) ~ .lever:hover::before {
+    transform: scale(2.4);
+}
+
+input[type=checkbox]:checked:not(:disabled) ~ .lever:hover::before {
+  background-color: $switch-reaction-checked-hover-color;
+}
+
 // Switch active style
 input[type=checkbox]:checked:not(:disabled) ~ .lever:active::before,
 input[type=checkbox]:checked:not(:disabled).tabbed:focus ~ .lever::before {
-  transform: scale(2.4);
-  background-color: transparentize($switch-bg-color, .85);
+  background-color: $switch-reaction-checked-focus-color;
+}
+
+input[type=checkbox]:not(:disabled) ~ .lever:hover::before {
+  background-color: $switch-reaction-unchecked-hover-color;
 }
 
 input[type=checkbox]:not(:disabled) ~ .lever:active:before,
 input[type=checkbox]:not(:disabled).tabbed:focus ~ .lever::before {
-  transform: scale(2.4);
-  background-color: rgba(0,0,0,.08);
+  background-color: $switch-reaction-unchecked-focus-color;
 }
 
 // Disabled Styles
 .switch input[type=checkbox][disabled] + .lever {
   cursor: default;
-  background-color: rgba(0,0,0,.12);
+  opacity: 0.5;
```

Comment:
has this been test on ie11 ?

Reply:
@roiLeo good point!
I checked it and it looks good 
![IMG_20201212_135238_564](https://user-images.githubusercontent.com/1275813/101984364-4e7a0780-3c81-11eb-8d07-71f9d7eca8a3.jpg)